Parse/Internal/Command/ParseCommand.cs

```diff
@@ -71,11 +72,6 @@ internal class ParseCommand : HttpRequest {
       if (!string.IsNullOrEmpty(ParseClient.PlatformHooks.OSVersion)) {
         Headers.Add(new KeyValuePair<string, string>("X-Parse-OS-Version", ParseClient.PlatformHooks.OSVersion));
       }
-      if (!string.IsNullOrEmpty(ParseClient.MasterKey)) {
```

Comment:
this was here for a reason, IIRC for Cloud Code stuff. Have you verified that IntegrationTest still run?